src/components/Notification/Notification.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
```

Comment:
react-test-renderer is slowly going away ;)... Enzyme gives us similar functionality and more fine grained tests... 

see:
https://github.com/patternfly/patternfly-react/pull/258